.github/pull_request_template.md

```diff
@@ -0,0 +1,49 @@
+## Title of this pull request
+
+* Link to discovery documentation (should contain all the planning and requirements for this work):
+   - See the `DISCOVERY.md` template at the root of the project
+* Link(s) to demo pages where this element can be viewed:
+
+---
+
+### What has changed and why
+Summarize files edited as part of this MR along with a brief description of what was changed/why.
+
+*
+
+### Testing instructions
+Be sure to include detailed instructions on how your update can be tested by another developer.
+
+1. 
+
+#### Browser requirements
+Your component should work in all of the following environments:
+
+- [ ] Latest 2 versions of Edge
+- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
+- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
+- [ ] Firefox 60.7.2 (or latest version for Red Hat Enterprise Linux distribution)
+- [ ] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
+- [ ] Latest 2 versions of Safari
+- [ ] Galaxy S9 Firefox
+- [ ] iPhone X Safari
+- [ ] iPad Pro Safari
+- [ ] Pixel 3 Chrome
+
+Your repository infrastructure updates should work for at least:
+- [ ] Node v8.x
```

Comment:
@mwcz Can you confirm these minimum versions for the infrastructure work?

Reply:
hub doc instructions from @mwcz  say
```
 (requires node v. 7 or higher)
```